#!/usr/bin/env python
PACKAGE = "common_front_end"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("fast_threshold",   int_t,   0, "threshold on difference between intensity of the central pixel and pixels of a circle around this pixel",  20, 5, 100)
gen.add("do_nms",   bool_t,   0, "if true, non-maximum suppression is applied to detected corners (keypoints)", True)
gen.add("skip_level0",   bool_t,   0, "detect features at top level of image pyramid", False)

exit(gen.generate(PACKAGE, "rslam_desktop", "FAST"))
